spec.html

```diff
@@ -3138,6 +3148,29 @@ <h1>Await Rejected Functions</h1>
         </emu-clause>
       </emu-clause>
 
+      <emu-clause id="yield" aoid="Yield">
+        <h1>Yield</h1>
+
+        <p>Algorithm steps that say</p>
+
+        <emu-alg>
+          1. Let _completion_ be Yield(_value_).
+        </emu-alg>
+
+        <p>mean the same thing as:</p>
+
+        <emu-alg>
+          1. Let _generatorKind_ be ! GetGeneratorKind().
+          1. If _generatorKind_ is ~async~, then
+            1. Let _completion_ be AsyncGeneratorYield(_value_).
```

Comment:
should this have a `?` or a `!`?

Reply:
no, the completion is passed to the caller.